src/constbuffer_array_splitter.c

```diff
@@ -208,3 +208,248 @@ CONSTBUFFER_ARRAY_HANDLE constbuffer_array_splitter_split(CONSTBUFFER_ARRAY_HAND
 
     return result;
 }
+
+TARRAY(CONSTBUFFER_ARRAY_HANDLE) constbuffer_array_splitter_split_to_array_of_array(CONSTBUFFER_ARRAY_HANDLE buffers, uint32_t max_buffer_size)
+{
+    TARRAY(CONSTBUFFER_ARRAY_HANDLE) result = NULL;
+
+    if (
+        /* Codes_SRS_CONSTBUFFER_ARRAY_SPLITTER_07_001: [ If buffers is NULL then constbuffer_array_splitter_split_to_array_of_array shall fail and return NULL. ]*/
+        buffers == NULL ||
+        /* Codes_SRS_CONSTBUFFER_ARRAY_SPLITTER_07_002: [ If max_buffer_size is 0 then constbuffer_array_splitter_split_to_array_of_array shall fail and return NULL. ]*/
+        max_buffer_size == 0)
+    {
+        LogError("Invalid args : CONSTBUFFER_ARRAY_HANDLE buffers = %p, size_t max_buffer_size = %" PRIu32,
```

Comment:
>2, [](http://example.com/codeflow?start=105&length=2)

Let's have a "" at the end. It clearly shows where the log line ends and where the args begin

Reply:
added, here and everywhere else